cluster-autoscaler/Dockerfile

```diff
@@ -15,6 +15,10 @@
 FROM gcr.io/google-containers/ubuntu-slim:0.8
 MAINTAINER Marcin Wielgus "mwielgus@google.com"
 
+RUN apt-get update && apt-get install --yes ca-certificates \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
To reduce image size to almost half. The size becomes 65MB if we omit `apt-get clean` and `rm`
![image](https://cloud.githubusercontent.com/assets/22009/25854708/f002efbc-350b-11e7-975c-1b2d8ad737d0.png)